### DISCUSSION TITLE
Nordigen, cleaning description/expense account on import or with rules? How?

### DISCUSSION THREAD

**Discussion by jaska087** [firefly-iii/firefly-iii#7397] [Firefly III] [answered]
My bank does not provide end to end id so the description for transactions are garbled up. (also missing some important tags to categorize transactions easily ;) )

"K-market Ii ))))EndToEndID: NOTPROVIDED " with expense account "K-market Ii ))))"
With my credit card (different bank) it's like this 
"K-market Ii Ii FI" with expense account "K-market Ii"

How could I remove the "))))EndToEndID: NOTPROVIDED" part from the description and " ))))" from expense account ?

I've read about the regex commit but haven't found any documents around this. I'd rather not create automatic rule for every store I visit.

Anyway love the software and got into auto import recently ;D

**Comment by JC5** [comment, marked as answer]
I'm sorry, this is not something that can be easily handled by rules. You could look into webhooks, but that may be on another level.
